renderer/components/organisms/AddProjectModal/AddProjectModal.js

```diff
@@ -21,12 +21,26 @@ class ModalContainer extends React.Component {
 		}
 
 		this.onType = this.onType.bind(this)
+		this.validateCheck = this.validateCheck.bind(this)
 	}
 
 	onType(e) {
 		this.setState({ [e.target.name]: e.target.value })
 	}
 
+	validateCheck() {
+		if(this.state.projectTitle !== '') {
```

Comment:
Beter iets als `.length > 2` denk ik? Wat vind jij?

Reply:
We hebben inderdaad daar nog geen rules voor. Zouden we moeten afspreken?😃